rpgJavaInterpreter-core/src/test/resources/EVALVARSNUMS.rpgle

```diff
@@ -0,0 +1,23 @@
+     D VAL             S              3  0
+     D RES             S              3  0
+
+     C                   EVAL      VAL=10
+
+     C                   EVAL      RES = 24 * VAL + 6
+     C     RES           DSPLY
+
+     C                   EVAL      RES=24 * VAL + 6
+     C     RES           DSPLY
+
+
+     C                   EVAL      RES=24* VAL + 6
+     C     RES           DSPLY
+
+     C                   EVAL      RES = 24* VAL + 6
+     C     RES           DSPLY
+
+     C                   EVAL      RES=24*VAL+6
+     C     RES           DSPLY
+
+     C                   EVAL      RES=24*VAL+6
```

Comment:
Just for curiosity, have you tried RES=24 *VAL + 6

Reply:
> Just for curiosity, have you tried RES=24 *VAL + 6

Yep, `*VAL` is considered like variable.